main.js

```diff
@@ -39,6 +44,10 @@ class MyVbus extends utils.Adapter {
         this.on('unload', this.onUnload.bind(this));
     }
 
+    async configIsValid(config) {
+        return ( '' !== (config.vbusPassword ) && ( (config.connectionIdentifier.match(ipformat) || config.connectionIdentifier.match(fqdnformat)) ) ) ;
+    }
+
```

Comment:
Hier müsste  in das ||  auch noch serialformat und vbusioformat abgefragt werden, sonst werden serielle und vbus.net Verbindungen abgewürgt.

Reply:
Baue ich noch ein. Kein Problem. Ich gebe zu, dass ich erst einmal auf meine Testumgebung fixiert war. ;-)

Reply:
Ist erledigt.